docs/code_examples/supervisord.conf

```diff
@@ -0,0 +1,5 @@
+[program:errbot]
+directory=/etc/errbot
+command=/path/to/my/virtualenv/bin/errbot
+autostart=true
+autorestart=true
```

Comment:
As mentioned, I'd use [my version](https://gist.github.com/zoni/9f5bb3043c91ab8dafd6a049ee1ce47d) instead. Most notably it makes sure supervisor sends the correct signal and ensures UTF-8 is used as locale.